spring-cloud-stream-binder-kafka/pom.xml

```diff
@@ -58,7 +69,7 @@
 		<dependency>
 			<groupId>org.springframework.integration</groupId>
 			<artifactId>spring-integration-kafka</artifactId>
-			<version>${spring-integration-kafka.version}</version>
+			<version>2.1.1.BUILD-SNAPSHOT</version>
```

Comment:
Right, that's correct. The main question here why don't do that (`2.1.1.BUILD-SNAPSHOT`) in the root pom, but just only here in the target project?
Feels like inconsistency.